src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -72,6 +78,15 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
   /** Stream that emits whenever the active item of the list manager changes. */
   change = new Subject<number>();
 
+  /**
+   * Predicate function that determines items that should be skipped by the list key manager.
+   * By default, disabled items are skipped by the key manager.
+   */
+  skipPredicate(predicateFn: (item: T) => boolean): this {
```

Comment:
Needs an `@param` for the `predicate`. Also it might be better to call it `predicate`, because the consumer can see that it's a function by looking a the type.

Reply:
:+1: